packages/locales/src/nl.ts

```diff
@@ -58,7 +61,9 @@ export const templates = {
   se412fe8953013d92: 'Selecteer tenminste één optie.',
   se952fc198acfa288: 'Naar het begin van de tabel',
   see63aaad45b1b116: 'status',
+  sefa2cd5631cae9d8: 'Stap omhoog',
   sf1ec4acb8d744ed9: 'Mededeling',
   sf677da98fa27f9b6: 'Meer links',
-  sf7290005be5beae6: str`${0}, pagina`
+  sf7290005be5beae6: str`${0}, pagina`,
+  sfc5e976d64c3fb31: 'Voer een geldig nummer in.'
```

Comment:
```suggestion
  sfc5e976d64c3fb31: 'Voer een geldig getal in.'
```
'nummer' is more for when it's digits used as identifier (bank account number, postal code etc) 'getal' is more when the value has meaning as a quantity

https://www.vlaanderen.be/team-taaladvies/taaladviezen/cijfer-getal-nummer